packages/app/src/store/studio-store.ts

```diff
@@ -319,18 +385,23 @@ export const useStudioStore = defineStore('studioRecorder', {
     reset () {
       this.stop()
 
+      this.restoreGrepSettings()
+
       this.logs = []
       this.url = undefined
       this._hasStarted = false
       this._currentId = 1
       this.isFailed = false
       this.showUrlPrompt = true
       this._isStudioCreatedTest = false
+      this._originalGrepSettings = {}
 
       this._maybeResetRunnables()
     },
 
     cancel () {
+      this.restoreGrepSettings()
```

Comment:
I don't think this is needed since `reset` is called which restores the grep settings.

Reply:
ah good catch